docs/devices.md

```diff
@@ -8,10 +8,11 @@ layout: page
 **Solaar** will detect all devices paired with your receiver, and at the very
 least display some basic information about them.
 
-At this moment, all [Unifying][unifying] Receiver are supported (devices with
-USB ID `046d:c52b` or `046d:c532`), but only some newer Nano Receiver (devices
-with USB ID `046d:c52f` and `046d:c52b`). You can check your connected Logitech
-devices by running `lsusb -d 046d:` in a console.
+At this moment, all [Unifying][unifying] receivers are supported (devices with
+USB ID `046d:c52b` or `046d:c532`), but only some newer Nano receivers (devices
+with USB ID `046d:c52f`, `046d:c52b`, and `046d:c534`). You can check your connected Logitech
```

Comment:
Okay, I noticed this is missing the lightspeed (gaming) receivers.

Reply:
OK, I added a bit about these receivers.  I can't be very specific as I don't know that much about how they work.

Reply:
Can you remove this sentence? This will show all Logitech devices.

Reply:
Github is being stupid and not showing my comment where it is supposed to. I am referring to the `lsusb` sentence.